### ISSUE TITLE
Add more to the Downloads page in regards to Installation

### ISSUE THREAD

**Issue by trosel** [Raku/raku.org#73]
https://perl6.org/downloads/

There should be a canonical way to install and upgrade rakudo star. The more detail we give here, the easier it will be to help troubleshoot installation issues.

- Have separate headings for each major platform
    - Mac section should discuss homebrew and whether that is a recommended method
    - Explain where things are installed and how to add them to your path
- Do not include rakudobrew if it continues to be fiddly
- Modern languages almost require a package manager of some sort. Include explanation / link to zef on this page?
- Most of all, assume that the user is a beginner

**Comment by zoffixznet** [comment]
Worth mentioning we have the downloads page on Rakudo: http://rakudo.org/how-to-get-rakudo/ along with a ticket for its improvement: https://rt.perl.org/Ticket/Display.html?id=130500#ticket-history

**Comment by WildYorkies** [comment]
@zoffixznet Good point. Ideally, the perl6 website would just link to each implementation's download page. As there's only one main one right now, it makes it a bit more closely coupled with the perl6.org site

**Comment by tbrowder** [comment]
I would change the "Download" to "Intall" and have it give clearer options than are available on the Rakudo site. I would change the "Radudo" button to a vertical list of four (or three buttons horizontally and one below):

    Windows   [link to rakubrew.org page]
    Mac Os    [link to rakubrew.org page]
    Linux     [link to rakubrew.org page]
    Other options [link to rakudo.org page]

**Comment by patrickbkr** [comment]
There are several factors that lead to the Downloads page on raku.org (and also rakudo.org):

- Keep the distinction of Raku and Rakudo up. Be clear about what is what to make a good as possible job of not confusing users. (That's why everything relating to downloads of a software was kept out of raku.org.)
- Don't favor one way of installing Rakudo over another. This is really difficult. But the rakudo.org/downloads tries its best to give an impartial overview of the different options. Putting Rakubrew on raku.org would give it a huge visibility advantage compared to all other installation options.
- Rakubrew (previously named Rakudobrew) has the goal of also supporting other Raku implementations once they show up. But it can't promise to keep up with that goal. So its not guaranteed to be an impartial installation method of Raku implementations.

The Downloads pages on raku.org and rakudo.org have historically been a big mess, both offering some installation options, none giving a comprehensive overview. We should make sure we're not drifting in that direction again.

I do notice the complexity in how a unknowing beginner has to navigate our pages to get to a download. But we need to conciously decide to give up on the above goals before introducing changes that violate them.

Having written all the above: I do not mean to blindly block all changes to the Downloads pages. What we have now is not perfect. We should try to improve.

---

I don't have strong emotions wrt "Downloads" vs "Installation". Can we get more opinions on this?

**Comment by tbrowder** [comment]
I have started a PR.

**Comment by tbrowder** [comment]
Coke, see closed PR #183

**Comment by coke** [comment]
@tbrowder : Can I ask, what's the rationale for switching from "Download" to "Install" - what problem specifically are you trying to solve with the nomenclature change?

**Comment by tbrowder** [comment]
When I was looking for a good method to install Raku and Zef with the least effort for a new person, I thought the distinction between downloads and installation methods, subtle though it may be, would be more appropriate for Windows users. So the PR changes come from that: in my mind, a more natural break into categories of installation methods and easier to select a suitable one.

**Comment by patrickbkr** [comment]
> @tbrowder : Can I ask, what's the rationale for switching from "Download" to "Install" - what problem specifically are you trying to solve with the nomenclature change?

Given that on raku.org no downloads are actually provided, "Installation" seems a bit less of a lie. So I tentatively approve of changing this.

(Anecdote: Emotionally I prefer "Download" to "Installation". I think the reason is, that I associate "Download" with "open source software" and "effortless install". While I associate "Installation" with "enterprise commercial software" and "many click throughs, walls of text, registration forms, agree to terms, ..." (aka "Can you please just give me the download?"))

**Comment by coke** [comment]
This has been reworked in the new site.

**Comment by tbrowder** [comment]
Great--thanks@!

**Comment by librasteve** [comment]
@tbrowder thanks for inspiring the general shape of the new raku.org site install page! (per your comment higher uo in this thread...)
